README.md

```diff
@@ -10,69 +10,66 @@ The Pi-hole[®](https://pi-hole.net/trademark-rules-and-brand-guidelines/) is a
-    E.g. guides are in `docs/guides`
-- Create the file using a URL friendly filename.
-    E.g. `docs/guides/url-friendly.md`
-- Edit your document using Markdown, there are loads of resources available for the correct syntax.
+### Testing Your Changes Locally
 
-### Testing your changes
+It's advised to review your changes locally before committing. Use the `mkdocs serve` command to live preview your changes on your local machine.
 
-When working on this repo, it is advised that you review your changes locally before committing them. The `mkdocs serve` command can be used to live preview your changes (as you type) on your local machine.
-
-Please make sure you fork the repo and change the clone URL in the example below for your fork:
-
-- Linux Mint / Ubuntu 18.04 LTS / 19.10 / 20.04 LTS:
-    - Preparations (only required once):
+**General Steps:**
+1. Fork the repository into your own Github account.
+2. Clone your fork and navigate into it. You need only do this once:
 
     ```bash
     git clone https://github.com/YOUR-USERNAME/docs
     cd docs
-    sudo apt install python3-pip
-    pip3 install -r requirements.txt
     ```
 
-    - Running the docs server:
+3. Install dependencies:
 
-    ```bash
-    mkdocs serve --dev-addr 0.0.0.0:8000
-    ```
+    - **Linux Mint / Ubuntu:** (18.04/20.04/22.04 LTS/19.10 and up)
 
-- Fedora Linux instructions (tested on Fedora Linux 28):
-    - Preparations (only required once):
+        ```bash
+        sudo apt install python3-pip
+        pip3 install -r requirements.txt
+        ```
 
-    ```bash
-    git clone https://github.com/YOUR-USERNAME/docs
-    cd docs
-    pip install --user -r requirements.txt
-    ```
+    - **Fedora Linux:** (tested on Fedora Linux 28)
 
-    - Running the docs server:
+        ```bash
+        pip install --user -r requirements.txt
+        ```
+
+4. Run the local docs server:
 
     ```bash
     mkdocs serve --dev-addr 0.0.0.0:8000
     ```
 
-- Docker instructions:
-    - One-shot run:
+**Docker Instructions:**
+For a one-shot run with Docker, use the following command:
 
-    ```bash
-    docker run -v `pwd`:/opt/app/ -w /opt/app/ -p 8000:8000 -it nikolaik/python-nodejs:python3.7-nodejs16 \
-      sh -c "pip install --user -r requirements.txt && \
-      /root/.local/bin/mkdocs build && \
-      npm ci && \
-      npm test && \
-      /root/.local/bin/mkdocs serve --dev-addr 0.0.0.0:8000"
-    ```
+```bash
+docker run -v $(pwd):/opt/app/ -w /opt/app/ -p 8000:8000 \
+  -it nikolaik/python-nodejs:python3.7-nodejs16 \
+  sh -c "pip install --user -r requirements.txt && \
+  /root/.local/bin/mkdocs build && \
+  npm ci && \
+  npm test && \
+  /root/.local/bin/mkdocs serve --dev-addr 0.0.0.0:8000"
+```
 
-After these commands, the current branch is accessible through your favorite browser at <http://localhost:8000>
+After running the above, your changes will be accessible through your favorite browser at http://localhost:8000.
```

Comment:
Markdownlint doesn't like bare URL in the text.

The previous code (`<http://localhost:8000>`) was generating a markdown link.

Please, revert this change.